addons/account/wizard/account_move_send.py

```diff
@@ -252,16 +252,9 @@ def _compute_checkbox_send_mail(self):
     def _compute_send_mail_extra_fields(self):
         for wizard in self:
             wizard.display_mail_composer = wizard.mode == 'invoice_single'
-            wizard.send_mail_warning_message = False
-
             invoices_without_mail_data = wizard.move_ids.filtered(lambda x: not x.partner_id.email)
             wizard.send_mail_readonly = invoices_without_mail_data == wizard.move_ids
-
-            if wizard.mode == 'invoice_multi' and wizard.checkbox_send_mail and invoices_without_mail_data:
-                wizard.send_mail_warning_message = _(
-                    "The partners on the following invoices have no email address, "
-                    "so those invoices will not be sent: %s",
-                    ", ".join(invoices_without_mail_data.mapped('name')))
+            wizard.send_mail_warning_message = 'missing_info' if invoices_without_mail_data else False
```

Comment:
>**Claire**: 
> 1. Maybe it's worth changing the send_mail_warning_message to a fields.Selection then ?
> 2. This will also make the warning visible even if checkbox_send_mail is False which we don't want.

Reply:
1. In stable, yes, we should / could. But I knew this wasn't gonna be merged before the freeze.
2. If no partner has the email set, the checkbox is readonly and I still want to know why I cannot mark it.
So I changed the condition a bit, see if it suits the case better.

![image](https://github.com/odoo/odoo/assets/1665365/8d84767e-5ed4-4151-ab07-841a4e760227)